model/src/main/java/com/epam/esm/specification/SpecificationBuilder.java

```diff
@@ -0,0 +1,32 @@
+package com.epam.esm.specification;
+
+import com.epam.esm.entity.GiftCertificate;
+import com.epam.esm.entity.QueryParameters;
+import com.epam.esm.specification.impl.SearchByStringGiftCertificateSpecification;
+import com.epam.esm.specification.impl.SearchByTagGiftCertificateSpecification;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
```

Comment:
it's not a strict rule, but according common usage service annotation is used for service layer interfaces implementations that contains business logic only(the same for repository, for dao impl) other components that aren't related to both cases can be marked as Component that enabled autoscan search them, even if it serves the needs of business logic (e.g. all util classes)